### PR TITLE
Add Elastic APM

### PR DESCRIPTION
**Pull request by ian-leggett**
## Description of change
This adds Elastic APM (Application Performance Monitoring) to the node layer so we can monitor transactions from the FE - https://www.elastic.co/guide/en/apm/agent/nodejs/3.x/configuring-the-agent.html

## Test instructions
This shouldn't effect anything on the FE, the only way you can see this take effect is by logging into our Kibana account and you will need access.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 require('dotenv').config()
 
+require('elastic-apm-node').start({
+  serviceName: process.env.ES_APM_SERVICE_NAME,
+  secretToken: process.env.ES_APM_SECRET_TOKEN,
+  serverUrl: process.env.ES_APM_SERVER_URL,
```
Comment (ambrozic) [inline src/server.js:6]: do we need this variables added in vault?

Comment (paulgain) [inline src/server.js:6]: Shouldn't these also be defined in https://github.com/uktrade/data-hub-frontend/blob/master/src/config/envSchema.js?

Comment (mcbhenwood) [inline src/server.js:6]: According to [the docs](https://www.elastic.co/guide/en/apm/agent/nodejs/current/express.html#express-advanced-configuration), if you set the env vars `ELASTIC_APM_SERVICE_NAME`, `ELASTIC_APM_SECRET_TOKEN` and `ELASTIC_APM_SERVER_URL` then you can just start APM without any args like this:
```javascript
require('elastic-apm-node').start()
```

File: src/server.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 require('dotenv').config()
 
+require('elastic-apm-node').start({
+  serviceName: process.env.ES_APM_SERVICE_NAME,
```
Comment (mcbhenwood) [inline src/server.js:4]: When configuring the env vars can we make sure the service name is 'datahub-fe' please. Thanks.

File: src/server.js [hunk 3]
```diff
@@ -1,5 +1,10 @@
 require('dotenv').config()
 
+require('elastic-apm-node').start({
```
Comment (mcbhenwood) [inline src/server.js:3]: I note that in the [DIT docs](https://readme.trade.gov.uk/docs/howtos/app-metrics.html), Mateusz is asking for us to set the SERVER_TIMEOUT to 20 seconds as standard also

Comment (ian-leggett) [reply src/server.js]: I have now set this to 20 secs as per documentation

File: src/server.js (hunk 2 above)
Comment (ian-leggett) [reply src/server.js]: All done

File: src/server.js (hunk 1 above)
Comment (ian-leggett) [reply src/server.js]: All done

Comment (ian-leggett) [reply src/server.js]: Good spot I forgot about this.

Comment (ian-leggett) [reply src/server.js]: I have added new ones into the Vault (datahub-fe) as I was referencing the BE ones before. If you check Vault you should see these in all environments.

**Pull request merged**